### PR TITLE
Use correct plurals in delete confirmation dialog

### PR DESCRIPTION
**Pull request by mymedia2**
It fixes plural form in the delete confirmation dialog: _these_. It also makes possible to get consistent translation here.

Please avoid joining of UI strings from small pieces like the case was here, as it makes harder to localize them.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -16,7 +16,10 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="appearance">Appearance</string>
     <string name="rename">Rename</string>
     <string name="confirm_delete">Confirm Delete</string>
-    <string name="do_you_really_want_to_delete_this_witharg">Do you really want to delete this %s?</string>
+    <plurals name="do_you_really_want_to_delete_this_document">
+        <item quantity="one">Do you really want to delete this document?</item>
```
Comment (gsantner) [inline app/src/main/res/values/strings.xml]: please have for both a separate string ID in strings.xml

Comment (gsantner) [inline app/src/main/res/values/strings.xml]: please have for both a separate string ID in strings.xml

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -16,7 +16,10 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="appearance">Appearance</string>
     <string name="rename">Rename</string>
     <string name="confirm_delete">Confirm Delete</string>
-    <string name="do_you_really_want_to_delete_this_witharg">Do you really want to delete this %s?</string>
+    <plurals name="do_you_really_want_to_delete_this_document">
```
Comment (gsantner) [inline app/src/main/res/values/strings.xml]: please move plurals to arrays.xml and refer both strings by string ID. See others.

Comment (mymedia2) [reply app/src/main/res/values/strings.xml]: This is wrong and causes troubles with translation into languages other than English where rules for varying numerals are different. Please do not do that anymore and fix remaining usage places of plural strings.

Comment (gsantner) [reply app/src/main/res/values/strings.xml]: You didn't get what I mean I guess.

```
    <plurals name="documents" translatable="false">
        <item quantity="one" translatable="false">@string/document</item>
        <item quantity="other" translatable="false">@string/documents</item>
    </plurals>
```

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (mymedia2) [reply app/src/main/res/values/strings.xml]: this indivisible string resource should have the single ID

Comment (gsantner) [reply app/src/main/res/values/strings.xml]: I don't want anything but key=value in strings.xml. Other cases have proofed not to work good with translation tools & services.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (mymedia2) [reply app/src/main/res/values/strings.xml]: You mark this resource as untranslatable and provide only two options, "one" and "other". That's not enough.

Comment (gsantner) [reply app/src/main/res/values/strings.xml]: Provide as many as you think needed. But use the said architecture. Thats the point.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (mymedia2) [reply app/src/main/res/values/strings.xml]: Ok, I can put the resource to other file, but `translatable="false"` shouldn't apply to it. Crowdin supports plurals resources.

Comment (gsantner) [reply app/src/main/res/values/strings.xml]: But others don't. I want to depend neither github, crowdin nor both.

**Comment by gsantner** [comment]
I've decided against this. I will reword that to just say along "delete following file(s)", keeping it simple.

**Pull request closed without merging**